src/components/KeyPad/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, memo } from 'react';
+import { Row } from '~components/Row';
+import { Column } from '~components/Column';
+import { FlatList, View } from 'react-native';
+import { leftButtonsLabels, rightButtonsLabels } from '~constants/buttonsLabels/buttonsLabels';
+import { styles } from './style';
+import { Button } from '~components/Button';
+import { KeyPadType } from '~components/KeyPad/type';
+
+export const KeyPad: FC<KeyPadType> = memo(({ onPressHandler }) => {
+    return (
+        <View style={styles.rootContainer}>
+            <Row>
+                <View style={styles.leftButtonsContainer}>
+                    {leftButtonsLabels.map(({ title, shape, type }) => (
```

Comment:
Why you don't use flatlist here?

Reply:
Because Flatlist doesnt support flexWrap: wrap